source/includes/etl_reports/_etl_lead_attribution.md

```diff
@@ -28,8 +53,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 }'
 ```
 
-> The format of the resulting csv file is:
+### Response Data Details;
```

Comment:
This was pointed out earlier and marked as resolved, but the ; character is still present.